tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/PetStoreClient.java

```diff
@@ -39,6 +41,30 @@ public PetStoreClient(String username, String password, TlsContextFactory tlsCon
         return config.getPets();
     }
 
+    @Override
+    public boolean isXA()
+    {
+        return false;
```

Comment:
I commented about this before, but remember another thing: a XA resource can be also used as non XA resource. Be sure to cover that scenario